starter-code/index.js

```diff
@@ -1,10 +1,34 @@
-var SortedList = function() {};
-
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
+var SortedList = function () {
+  var self = this;
+
+  self.items = [];
+  self.length = self.items.length;
```

Comment:
This would work, but better:
```
this.length = 0;
```
You are simulating your own data structure with its own properties.